src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <>
```

Comment:
looks redundant here 

https://react.dev/reference/react/Fragment#usage

Reply:
I wrote
`const partner = person.sex === 'm' ? 'wife' : 'husband'`
because the last time @nazarmatsevych checked my assignment I was asked to do this. Not sure how to solve it without "return"